packages/clerk-js/src/ui/signUp/SignUpStart.tsx

```diff
@@ -302,7 +302,7 @@ function _SignUpStart(): JSX.Element {
       </Control>
     ) : null;
 
-  const atLeastOneFormField = nameField || usernameField || emailAddressField || phoneNumberField || passwordField;
+  const showFormFields = userSettings.hasValidAuthFactor || (!oauthOptions.length && !oauthOptions.length);
```

Comment:
🔧 
I think this should be checking for web3 options, the `(!oauthOptions.length && !oauthOptions.length)` check is probably wrong

Reply:
wrong copy/paste when backtesting the fix :P